torchtext/utils.py

```diff
@@ -1,6 +1,8 @@
 import six
 import requests
 import csv
+import math
```

Comment:
Take a look at details from travis build. The log suggests this line is not used, so it should be deleted.

Reply:
I'm sorry, my fault. Forgot to remove this import statement.